boards/arm/npcx9m6f_evb/doc/index.rst

```diff
@@ -0,0 +1,98 @@
+.. _npcx9m6f_evb:
+
+NPCX9M6F_EVB
+###################
+
+Overview
+********
+
+The NPCX9M6F_EVB kit is a development platform to evaluate the
+Nuvoton NPCX9 series microcontrollers. This board needs to be mated with
+part number NPCX996F.
+
+.. image:: ./npcx9m6f_evb.png
+     :width: 800px
+     :align: center
+     :alt: NPCX9M6F Evaluation Board
+
+Hardware
+********
+
+- ARM Cortex-M4F Processor
+- 256 KB RAM and 64 KB boot ROM
+- ADC & GPIO headers
+- UART0 and UART1
+- FAN PWM interface
+- Jtag interface
+- Intel Modular Embedded Controller Card (MECC) headers
+
+Supported Features
+==================
+
+The following features are supported:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | reset and clock control             |
```

Comment:
The board DTS file also enables the following components:
PSL, PWM, ADC, eSPI, I2C, TACH
Plus power management is also supported.

Reply:
Done.